src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -70,7 +71,7 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
   trsLink: string;
   displayTextForButton: string;
   displayedColumns: string[] = ['name', 'role', 'affiliation', 'email', 'orcid_id'];
-  authors: (Author | OrcidAuthor)[];
+  authors: any[] = [];
```

Comment:
Can the elements of `authors` be of a type other than `Author` or `OrcidAuthor`?

Reply:
The elements should only be `Author` and `OrcidAuthor`. I was getting errors like `Type 'Author' is not assignable to type 'Author | OrcidAuthor'. Property 'orcid' is missing in type 'Author' but required in type 'OrcidAuthor'.` so I changed it to `any`. Checking again, the error disappears if I use the models from openapi so I changed it back